### PR TITLE
[Codegen] remove fp16 function override for cuda 

### PR DESCRIPTION
**Pull request by yzhliu**
override __w/o__ volatile does not have any effect as cuda already has the functionality.
override __w/__ volatile is not supported by NVRTC at the moment.
I suggest to disable the feature for now and enable later once we find the solution.
see https://discuss.tvm.ai/t/error-fp16-cuda-compilation-error/ for details.

@vinx13 @Hzfengsy @tqchen Please advise.

### CODE REVIEW

File: src/codegen/literal/cuda_half_t.h [hunk 1]
```diff
@@ -76,7 +77,7 @@ class TVM_ALIGNED(2) half {
   TVM_XINLINE explicit half(const uint8_t& value) { constructor(value); }
   TVM_XINLINE explicit half(const int32_t& value) { constructor(value); }
   TVM_XINLINE explicit half(const uint32_t& value) { constructor(value); }
-  TVM_XINLINE explicit half(const int64_t& value) { constructor(value); }
+  TVM_XINLINE explicit half(const long long& value) { constructor(value); }
```
Comment (vinx13) [inline src/codegen/literal/cuda_half_t.h]: why this change?

Comment (yzhliu) [reply src/codegen/literal/cuda_half_t.h]: do I need to `#include <cstdint>` for `int64_t` ?

Comment (vinx13) [reply src/codegen/literal/cuda_half_t.h]: it's okay to use `long long` as they are the same for cuda
